### PR TITLE
Update sustainability labels

### PR DESCRIPTION
**Pull request by BigDatalex**
The scraping run on '2022-08-12' yielded about 18k `UNKNOWN `certificates for Zalando and about 700 for Otto. For Amazon numbers see #88 This PR focuses on decreasing this number and adding a few more improvements. In detail the following things are covered:

- Update amazon, asos, zalando and otto labels. The shops mostly changed the naming of already existing labels, but added also a few new ones.
- Add logging function as part of `sustainability_labels_to_certificates`, which logs unknown certificates. So far we logged only those for zalando. Now amazon and otto are also covered.
- Products without any label are no more assigned to UNKNOWN, instead they are excluded.
- Already added extraction of the (new) EU Energy label for amazon in preparation of new household categories of #83 
- There were some issues with the amazon description extraction if an empty string was retrieved for `productDescription`, but there were actually `feature-bullets`. This is fixed now, by using the feature-bullets instead and if both (feature-bullets and productDescription) are available they are concatenated as suggested in #81

### CODE REVIEW

File: extract/extract/utils.py [hunk 1]
```diff
@@ -47,47 +49,60 @@ def get_product_from_JSON_LD(json_ld: List[Any], else_return: Any = {}) -> Any:
...
+    for certificate_string, certificate in result.items():
+        if certificate is None:
+            if certificate_string in certificate_mapping.keys():
+                result.update({certificate_string: certificate_mapping[certificate_string]})
+            else:  # if certificate_string can not be mapped, assign UNKNOWN and create log message
+                result.update({certificate_string: CertificateType.UNKNOWN})  # type: ignore[attr-defined] # noqa
+                logger.info(f"unknown sustainability label: {certificate_string}")
 
-    return sorted(set(result)) or [CertificateType.UNKNOWN]  # type: ignore[attr-defined]
+    return sorted(set(list(result.values())))  # type: ignore[attr-defined]
```
Comment (se-jaeger) [inline extract/extract/utils.py:93]: remove the `# type:` comment

Comment (se-jaeger) [reply extract/extract/utils.py:93]: It's not necessary anymore, is it?

Comment (BigDatalex) [reply extract/extract/utils.py]: Yes, that's right.

**Review by se-jaeger — APPROVED** [review]
LGTM 🚀

**Pull request merged**